contracts/examples/ping-pong-egld/wasm/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ping-pong-wasm"
+version = "0.0.1"
+authors = [ "you",]
```

Comment:
you can add your name here too, or remove the field

Reply:
done